.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
 
 after_success:
   - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker build -t echosoft/taxi .
-    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-    docker push echosoft/taxi:latest
-    fi
\ No newline at end of file
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
```

Comment:
where are these being set? do we have a private wiki page or something to get setup?

Reply:
In travis environmental variables